### PR TITLE
Dasepho rubbots

### PR DESCRIPTION
**Pull request by Dasepho**
Basic rub-bot explanation with diagrams of the stringing process, which is most in need of documenting

I haven't previewed the site with these changes - can I? Does Netlify publish to something like [https://sro.netlify.app/dasepho_rubbots](https://sro.netlify.app/dasepho_rubbots)?

### CODE REVIEW

**Comment by Dasepho** [comment]
> I haven't previewed the site with these changes - can I? Does Netlify publish to something like https://sro.netlify.app/dasepho_rubbots?

Figured it out, images and link should be fixed

**Comment by trickeydan** [comment]
https://deploy-preview-56--sro.netlify.app/

File: docs/kit/outreach/rub-bots.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Rub-bots
+---
+
+The Rub-bots are a set of physical robots constructed in July/August
+2021 to facilitate [Smallpeice 2021](/events/history/2021). For that competition, they were called the 'Forklift-bots' to keep with the theme of the game. After that, they became the 'rub-bots' - named to reflect the fact that they were designed to perfectly fit inside the 35L Really Useful Boxes ('RUBs')
```
Comment (tpoomlmly) [inline docs/kit/outreach/rub-bots.md:6]: the 35L rubs are actually too narrow to fit a rub-bot, but they fit perfectly in the 18L ones
```suggestion
2021 to facilitate [Smallpeice 2021](/events/history/2021). For that competition, they were called the 'Forklift-bots' to keep with the theme of the game. After that, they became the 'rub-bots' - named to reflect the fact that they were designed to perfectly fit inside the 18L Really Useful Boxes ('RUBs')
```

**Comment by tpoomlmly** [comment]
I love the string diagrams

**Comment by tpoomlmly** [comment]
Is there a schematic for how the chassis goes together?

**Comment by tpoomlmly** [comment]
A BoM and STLs would be useful too

**Review by Dasepho — COMMENTED** [review]
> Is there a schematic for how the chassis goes together?

Not yet, but I agree it would be useful for me to add one

> A BoM and STLs would be useful too

Agreed, BOM should be easy - I did that already for purchasing. STLs (and probably full CAD as step file) feel like they should be in a roboticsoutreach/rubbots repo or something? I can clean up my working directory and push it. Would probably want to keep the documentation all together tho, so I'd link from roboticsoutreach/rubbots to the kb.roboticsoutreach.org/kit/outreach/rubbots page

I'll likely work on it Tuesday/Wednesday, I'm busy this weekend

File: docs/kit/outreach/rub-bots.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+The design was intended to be robust so the robots would last long enough to be used for many future events, as demo-bots and to be offered to participants for outreach events.
+
+The bots have:
+- 2 wheel differential drive
+- a servo-driven linear gripper
+- a forward-facing ultrasound distance sensor
```
[the remark this answers is not in the document]
Comment (Dasepho) [reply docs/kit/outreach/rub-bots.md:14]: Fixed by 4943daa512f03994b265a6576e60f02320ca9b58

**Review by tpoomlmly — APPROVED** [review]

**Pull request merged**